### PR TITLE
Migrate all tests to specs2

### LINKED ISSUE #4: Migrate tests to specs2

### PR DESCRIPTION
**Pull request by jcranky**
closes #4

### CODE REVIEW

File: ScalaMarkdown/test/org/fusesource/scalamd/MarkdownSpec.scala [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-import org.scalatest.prop.TableDrivenPropertyChecks
-import org.scalatest.{Matchers, PropSpec}
+import org.specs2.Specification
 
 import scala.io.Source
 
-class MarkdownSpec extends PropSpec with Matchers with TableDrivenPropertyChecks {
+class MarkdownSpec extends Specification {
+  def is =
+    s2"""
```
Comment (tr4rex) [inline ScalaMarkdown/test/org/fusesource/scalamd/MarkdownSpec.scala:9]: is it a typo?

Comment (jcranky) [reply ScalaMarkdown/test/org/fusesource/scalamd/MarkdownSpec.scala:9]: No, it is _specs2_ string interpolation for running immutable specifications.

**Comment by jmhofer** [comment]
I guess you don't want to switch everything to `org.specs.Specification` at once? 😉

**Review by jmhofer — APPROVED** [review]

**Comment by jcranky** [comment]
@jmhofer no, would change too much XD - and that big test could perhaps be broken into smaller ones, but lets firstunblock people :)

**Comment by jcranky** [comment]
Now I'm just trying to understand why codacy is not checking PRs anymore

**Pull request merged**